CountBeans.js

```diff
@@ -0,0 +1,16 @@
+// Считаем "В". Не "В" тоже считаем
+
+const countChar = (string, letter = 'b') => {
+    let counter = 0;
+    string = string.toLowerCase();
+    
+    for(let i = 0; i < string.length; i++) {
+        if(string[i] == letter) {
+            counter++;
+        }
+    }
+    console.log(counter);
+}
+
+countChar('Bubble'); // CountBs из условия
+countChar('Not a bubble', 'l');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Varlyle/my-first-project/pullRequest?prid=7306921)
```suggestion
countChar("Not a bubble", 'l');
```